docs/serializer-formatter.rst

```diff
@@ -48,7 +48,7 @@ to Kafka. A ``SerializationException`` may occur during the send call, if the da
     import java.util.Properties;
 
     Properties props = new Properties();
-    props.put(ProducerConfig.BOOTSTRAP_SERVERS_CONFIG, "localhost:9092");
```

Comment:
This seems backwards. 9092 is the normal port for Kafka, which is what bootstrap.servers should be pointing to. I think the issue in the ticket was re: the consumer code, which is either just an accident or possibly when converting to new consumer the port wasn't changed. I think you might actually need to fix this part in a newer branch, it looks like this one doesn't have the new consumer example that the issue is referring to.